server/controllers/articles.js

```diff
@@ -214,6 +214,52 @@ class ArticleController {
       next(err);
     }
   }
+
+  /**
+  * Users can like an article
+  *
+  * @static
+  * @param {*} req
+  * @param {*} res
+  * @param {*} next
+  * @memberof ArticleController
+  * @return {json} Returns json object
+  */
+  static async like(req, res, next) {
+    try {
+      const me = req.user;
+      const { slug } = req.params;
+      const article = await findArticle({ slug });
```

Comment:
you should include the number of likes an article has in its attribute.

Please modify the `findArticle` method to include the sum of likes an article has.

Please only the sum not details

Reply:
done and dusted.